src/styles/blocks/content.scss

```diff
@@ -0,0 +1,51 @@
+.content {
+  display: flex;
+  justify-content: center;
+  position: relative;
+  width: $clock-size;
+  height: $clock-size;
```

Comment:
you can use `@mixin` for this. Since your circles are squares with border-radius, you can use only one mixin.

Reply:
Thankyou for advice, I did it.